src/client/visualizations/table/highlight.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017-2018 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from "react";
+import "./highlight.scss";
+
+interface HighlighterProps {
+  top: number;
+  left: number;
+}
+
+export const Highlighter: React.SFC<HighlighterProps> = ({ top, left }) => {
+  return <div className="highlight-cont">
+    <div className="highlight">
+      <div className="highlighter" key="highlight" style={{ top, left }} />
+    </div>
+  </div>;
```

Comment:
three element to render highlight? Is it possible to optimize dom node number?

Reply:
Yeah, seems excessive, but I don't want to touch it there.